### PR TITLE
Updates the "New Issue" behaviour to better redirect users.

### PR DESCRIPTION
**Pull request by stuartarchibald**
This uses the config.yml functionality to add links out to discourse
in the case of users with questions/feature discussion. It also
updates the bug report and feature request templates, removing the
suggestion to ask on gitter.im as users with more question like
problems should already have been intercepted.

### CODE REVIEW

**Comment by stuartarchibald** [comment]
Note there's a couple of TODOs where links need updating once discourse is sorted out. Am also wondering if the feature request template is needed? Perhaps feature request should go into discourse so they can be discussed and then an issue curated from the outcome?

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature Request
-about: Suggest an idea for Numba
+about: Tell us about something that it'd be good for Numba to support
```
Comment (esc) [inline .github/ISSUE_TEMPLATE/Feature_request.md:3]:
```suggestion
about: Tell us about something that would be good for Numba to support
```

Comment (esc) [reply .github/ISSUE_TEMPLATE/Feature_request.md:3]: Actually the sentence now sounds even weirder, maybe we need a different approach entirely.

Comment (stuartarchibald) [reply .github/ISSUE_TEMPLATE/Feature_request.md:3]: How about "Tell us about something you'd like to see supported in Numba" ?

Comment (esc) [reply .github/ISSUE_TEMPLATE/Feature_request.md:3]: That does sound more fluid, yes.

Comment (stuartarchibald) [reply .github/ISSUE_TEMPLATE/Feature_request.md:3]: Or even "Tell us about something in the Python language/NumPy you'd like to see supported in Numba" given that this is opening a feature request in the tracker, limiting the scope to simple "X is missing" type things might be a good idea as more involved stuff is already being redirected towards discourse?

Comment (gmarkall) [reply .github/ISSUE_TEMPLATE/Feature_request.md:3]: How about "Tell us about something in the Python language/NumPy you'd like Numba to support"?

Comment (stuartarchibald) [reply .github/ISSUE_TEMPLATE/Feature_request.md:3]: I like that! Thanks @gmarkall

Comment (stuartarchibald) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: e496e19  has this

**Comment by stuartarchibald** [comment]
Links to discourse topics are now updated given they now actually exist!

**Comment by stuartarchibald** [comment]
Thanks for reviewing @hameerabbasi

**Comment by stuartarchibald** [comment]
Thanks for the feedback all, merging!

**Comment by esc** [comment]
👍

**Pull request merged**